cfn-resources/project-ip-access-list/mongodb-atlas-projectipaccesslist.json

```diff
@@ -77,42 +104,16 @@
     "/properties/ProjectId",
     "/properties/Profile"
   ],
-  "properties": {
-    "AccessList": {
-      "items": {
-        "$ref": "#/definitions/accessListDefinition"
-      },
-      "type": "array",
-      "insertionOrder": false
-    },
-    "ProjectId": {
-      "description": "Unique 24-hexadecimal digit string that identifies your project.",
-      "type": "string"
-    },
-    "ListOptions": {
-      "$ref": "#/definitions/listOptions"
-    },
-    "TotalCount": {
-      "description": "Number of documents returned in this response.",
-      "type": "integer"
-    },
-    "Profile": {
-      "type": "string",
-      "description": "Profile used to provide credentials information, (a secret with the cfn/atlas/profile/{Profile}, is required), if not provided default is used",
-      "default": "default"
-    }
-  },
   "readOnlyProperties": [
-    "/properties/AccessList",
-    "/properties/AccessList/AwsSecurityGroup"
```

Comment:
I see that these props were added in #327. @Adelmar92 I am not sure why these props were added as Read Only. Users need to add new entries to the atlas access list using  `properties/AccessList`. if we make it read-only, they cannot use this field during the creation phase. What Am I missing? 🤔

Reply:
Hi Andrea! this is something we have to fix, AccessList souldnt be ReadOnly